internal/utils/aggregator.go

```diff
@@ -87,3 +116,25 @@ func Aggregate_data(df dataframe.DataFrame) dataframe.DataFrame {
 	df = dfGroups.Aggregation(columnsAggregationType, columnsToAggregate)
 	return df
 }
+
+func filter_valid_csv_records(main_df dataframe.DataFrame) (dataframe.DataFrame, int) {
```

Comment:
If I understand if a value is missing then its 0 and so we can compare that to GreaterEq 0, I just like to confirm there is no missing or nil value?
At the same time can we have tests showcasing valid/invalid data?

Reply:
We are filtering out/dropping the records where values are missing.

Reply:
Added test cases